### PR TITLE
chore(concept): #849 create concept

### PR DESCRIPTION
**Pull request by ds-crehm**
resolves eclipse-tractusx/traceability-foss#849

### CODE REVIEW

File: docs/concept/#849-notifications-to-multiple-bpns/#849-notifications-to-multiple-bpns.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+The data will be identical except for the "sendTo" and "sendToName" fields.
+If the data must be changed, the user can do it afterward in the "edit notification" view.
+
+#### Investigation creation
+When an investigation is created, multiple parts can be selected. Whenever a part from a different BPN is selected, the notification will be duplicated.
+The data will be identical except for the "sendTo", "sendToName" and "assetIds" fields. Each notification will only contain parts from the same BPN.
+If the data must be changed, the user can do it afterward in the "edit notification" view.
+
+#### Edit alerts
+When editing an alert, only one BPN is shown since alerts to multiple BPNs are separate duplicated alerts. Removing this BPN is not possible.
```
Comment (ds-mwesener) [inline docs/concept/#849-notifications-to-multiple-bpns/#849-notifications-to-multiple-bpns.md:49]: From my perspective it should not be allowed to edit elements which causing a creation of a new notification.

File: docs/concept/#849-notifications-to-multiple-bpns/#849-notifications-to-multiple-bpns.md [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+- In the modal the user has the chance to cancel or approve the process.
+
+# Out of scope
+- A hierarchical parent object, that ties all the duplicated notifications together. For now, the user must search/filter for all individual notifications if he wants to interact with all of them.
+In the future, the idea is to bundle all duplicated notifications in a parent object that can be searched/filtered for.
+
+# Concept
+## Backend
+
+#### Alert creation
```
Comment (ds-mwesener) [inline docs/concept/#849-notifications-to-multiple-bpns/#849-notifications-to-multiple-bpns.md:38]: The last iterations we made sure that there is almost no difference anymore between alerts and investigations. Why do we have different behaviours defined here?

File: docs/concept/#849-notifications-to-multiple-bpns/#849-notifications-to-multiple-bpns.md (hunk 1 above)
Comment (ds-crehm) [reply docs/concept/#849-notifications-to-multiple-bpns/#849-notifications-to-multiple-bpns.md]: Changed it to not be possible.

File: docs/concept/#849-notifications-to-multiple-bpns/#849-notifications-to-multiple-bpns.md (hunk 2 above)
Comment (ds-crehm) [reply docs/concept/#849-notifications-to-multiple-bpns/#849-notifications-to-multiple-bpns.md:42]: During the creation of investigations/alerts, there is a different behaviour for the BPN selection. That cannot be changed due to the nature of the use cases. So I described both scenarios, while keeping the process as similar as possible.

File: docs/concept/#849-notifications-to-multiple-bpns/#849-notifications-to-multiple-bpns.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+# Overview
+Instead of having "messages" that can be sent to multiple BPNs within one notification, multiple notifications must be created.
+Those can be individually edited and sent to the specific BPN.
+
+# Requirements
+- For alerts, when selecting multiple BPNs, a duplicated alert for each of the selected BPNs is created.
+- For investigations, when selecting parts from different BPNs, a duplicated investigation for each of the BPNs is created.
+- The data model is changed to make this process possible.
+- Notifications are tied to single BPNs and cannot be sent to multiple BPNs anymore.
+- BPN selection is possibly for alerts.
```
Comment (ds-mwesener) [inline docs/concept/#849-notifications-to-multiple-bpns/#849-notifications-to-multiple-bpns.md:26]:
```suggestion
- BPN selection is only possible for alerts.
```

File: docs/concept/#849-notifications-to-multiple-bpns/#849-notifications-to-multiple-bpns.md [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+  - Autocomplete filter -> Selection shows all matching BPNs (if there are more than 10, this selection is scrollable).
```
Comment (ds-mwesener) [inline docs/concept/#849-notifications-to-multiple-bpns/#849-notifications-to-multiple-bpns.md:27]: To be discussed.

File: docs/concept/#849-notifications-to-multiple-bpns/#849-notifications-to-multiple-bpns.md [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+  - Selected BPNs are shown as chips that can be removed.
```
Comment (ds-mwesener) [inline docs/concept/#849-notifications-to-multiple-bpns/#849-notifications-to-multiple-bpns.md:28]: Please link to the story which implements that feature

File: docs/concept/#849-notifications-to-multiple-bpns/#849-notifications-to-multiple-bpns.md [hunk 6]
```diff
@@ -0,0 +1,96 @@
...
+- BPN selection is disabled for investigations, instead the BPNs are "pulled" from the selected parts.
+- The frontend modals appear when a user is creating notifications and triggers a notification duplication.
+- In the modal the user has the chance to cancel or approve the process.
+
```
Comment (ds-mwesener) [inline docs/concept/#849-notifications-to-multiple-bpns/#849-notifications-to-multiple-bpns.md:32]: Available parts are additionally filtered by the bpns in the bpn input field.

File: docs/concept/#849-notifications-to-multiple-bpns/#849-notifications-to-multiple-bpns.md [hunk 7]
```diff
@@ -0,0 +1,96 @@
...
+The data will be identical except for the "createdFor" and "createdForName" fields.
+If the data must be changed, the user can do it afterward in the "edit notification" view.
+
+#### Investigation creation
+When an investigation is created, multiple parts can be selected. Whenever a part from a different BPN is selected, the notification will be duplicated.
+The data will be identical except for the "createdFor", "createdForName" and "assetIds" fields. Each notification will only contain parts from the same BPN.
+If the data must be changed, the user can do it afterward in the "edit notification" view.
+
+#### Edit notifications
+When editing notifications, only parts with the identical BPN as the "createdFor" BPN will be shown and the user may not add BPNs manually.
```
Comment (ds-mwesener) [inline docs/concept/#849-notifications-to-multiple-bpns/#849-notifications-to-multiple-bpns.md:82]: Please describe which fields can be edited.

**Review by ds-mwesener — CHANGES_REQUESTED** [review]
Please adress comments. Thank you

File: docs/concept/#849-notifications-to-multiple-bpns/#849-notifications-to-multiple-bpns.md (hunk 4 above)
Comment (ds-crehm) [reply docs/concept/#849-notifications-to-multiple-bpns/#849-notifications-to-multiple-bpns.md:28]: Will leave it in the concept for now. If it's not possible, we can either change the concept or directly change the PBIs.

File: docs/concept/#849-notifications-to-multiple-bpns/#849-notifications-to-multiple-bpns.md (hunk 5 above)
Comment (ds-crehm) [reply docs/concept/#849-notifications-to-multiple-bpns/#849-notifications-to-multiple-bpns.md:29]: This should be done in the frontend PBI. It's linked in the ticket description of the concept (#849)

File: docs/concept/#849-notifications-to-multiple-bpns/#849-notifications-to-multiple-bpns.md (hunk 6 above)
Comment (ds-crehm) [reply docs/concept/#849-notifications-to-multiple-bpns/#849-notifications-to-multiple-bpns.md:34]: Added this

**Review by ds-mwesener — APPROVED** [review]
Approved, we might think of a data format for the case id.

**Pull request merged**